README.md

```diff
@@ -66,434 +66,6 @@ This roster is seeded with approvers from the Client workgroups.
-
-## Post-release work
-
-### Homebrew updates
-
-#### homebrew-client
-
-After the client release, the [Homebrew tap](https://github.com/knative/homebrew-client) needs to be updated with the new release:
-
-* Copy `kn.rb` to the `kn@${PREV_RELEASE}.rb` with `$PREV_RELEASE` to be replace with the latest release (e.g. `0.19`).
-* In `kn@${PREV_RELEASE}.rb` replace `class Kn` with `class KnAT${PREV_RELEASE_DIGITS}`, e.g `class KnAT019` for an previous release `0.19`.
-* In `kn.rb`
-  - Replace the old version number in `v` with the released version (e.g. `v = "v0.20.0"`)
-  - Replace the `sha256` checksums with the values from the [client release page](https://github.com/knative/client/releases). The checksums have been released, too (e.g. [checksums.txt](https://github.com/knative/client/releases/download/v0.22.0/checksums.txt))
-
-Create a PR and merge the changes. Prow is not enabled for the homebrew repo, so the merge needs to be performed manually.
-
-#### homebrew-kn-plugins
-
-Similar to the client repo, the [client plugin's Homebrew repo](https://github.com/knative-sandbox/homebrew-kn-plugins) needs to be updated
-for the the plugins supported after their repos have successfully created a release.
-
-Currently the following plugins are available with their own formulas:
-
-* [kn-plugin-admin](https://github.com/knative-sandbox/kn-plugin-admin) is managed via the `admin.rb` formula
-* [kn-plugin-source-kafka](https://github.com/knative-sandbox/kn-plugin-source-kafka) is managed via `source-kafka.rb` formula
-* [kn-plugin-source-kamelet](https://github.com/knative-sandbox/kn-plugin-source-kamelet) is managed via `source-kamelet.rb` formula
-* [kn-plugin-quickstart](https://github.com/knative-sandbox/kn-plugin-quickstart/) is managed via `quickstart.rb` formula
-
-The artifact checksums can be found on the respective release pages. Once the
-PR is merged, [cut a new branch](#cut-the-branch). There is no automation after
-this, so you're done here.
-
-### Final checks & transition to the next release leads
-
-1. Add the release to the [release table](https://github.com/knative/community/blob/main/mechanics/RELEASE-SCHEDULE.md)
-
-1. Watch for the PR like [this one](https://github.com/knative/test-infra/pull/2670)
-to enable dot releases on the new releases and approve it. These are create by the Prow
-https://prow.knative.dev/?job=ci-knative-prow-jobs-syncer
-
-1. Send a PR like [this one](https://github.com/knative/community/pull/619) to
-grant ACLs for the next release leads, and to remove yourself from the rotation.
-Include the next release leads in the PR as a reminder.
-
-1. Send a PR like [this one](https://github.com/knative-sandbox/knobots/pull/152/files) to
-bump knobots auto release workflow to the next release.
+# Release Process
 
+[Instructions](./releasing.md) on how to release a `knative.dev` repository.
```

Comment:
Not a fan of the redirect here. Why not just have everything in the root?

Reply:
For me it makes sense, you can have a concrete guide for the release process without all the roster and release versions distracting